app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,18 @@
+//= require_tree ../stylesheets
```

Comment:
it's better to use import specific files. Easier to manage sequence and sometimes there are cases where some files should not be used in app.scss